Campus Density/DiningCells/DetailControllerHeaderCell.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  DetailControllerHeaderCell.swift
+//  Campus Density
+//
+//  Created by Ansh Godha on 26/06/20.
+//  Copyright © 2020 Cornell DTI. All rights reserved.
+//
+
+import UIKit
+
+class DetailControllerHeaderCell: UICollectionViewCell {
+
+    var place: Place!
```

Comment:
Personally feels like a bit much to give this component all of the information in the `Place` (which includes things like menus) compared to only the information it directly needs to display itself. But up to you I guess since this is easier and more straightforward

Reply:
Hmm ok will look into this